ocaml/typing/typecore.ml

```diff
@@ -334,6 +333,13 @@ let position_and_mode_default = {
   region_mode = None;
 }
 
```

Comment:
Thanks for writing a new record type. We should comment it as to how and where it's used.

Reply:
Done!